worker-x86_64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensuse:tumbleweed
+FROM opensuse/tumbleweed
```

Comment:
Thanks!

Actually it is not a typo. Time ago the official images were delivered to the docker hub standard library `library/opensuse:tumbleweed`. But, now the official openSUSE images are delivered to the repository that they own `library/opensuse/tumbleweed:latest`

Could you change the subject line of the commit?
e.g. `Use openSUSE's repository instead of docker's`

Reply:
Oh I assumed it was a typo, as in the other Dockerfile it's `opensuse/tumbleweed`, as it should. :smiley: